### PR TITLE
[ARTS-190] - UI tests around setting locale in a trial feature

### PR DESCRIPTION
**Pull request by sameerapurini**
## Description
The PR has UI tests written in landing page feature that cover the acceptance criteria of arts-190 ticket. Scenarios have been added to the existing landing page feature in order to check both default and set locale from the drop down menu.
Also updated is all the features with author of the feature.

### Changes
- ARTS-190- Setting locale in a trial

### CODE REVIEW

File: ui-tests/pages/authenticationPage.page.js [hunk 1]
```diff
@@ -8,6 +9,7 @@ class authenticationPage {
     get landingPageWelcomeMessage() { return $('//h1[text()="Welcome"]') }
     get selectYes() { return $('//input[@value="Yes"]') }
     get logOutButton() { return $('//div//button[text()="Logout"]') }
+    get localeLogOutButton() { return $('//div//button[text()="xLogoutx"]') }
```
Comment (mdp0) [inline ui-tests/pages/authenticationPage.page.js:12]: Would a parameter in the step definition to cope better with the changes in text? A step like "The label for the logout button reads {string}"

File: ui-tests/pages/landingPage.page.js [hunk 2]
```diff
@@ -1,12 +1,27 @@
...
     get loginButton() { return $('//button[@id="login_button"]') }
+    get defaultLocale() { return $('//span//div//select[@name="currentLocale"]//option[text()="English (UK)"]') }
+    get preferredLocale() { return $('//span//div//select[@name="currentLocale"]//option[@value="1: en-xx"]') }
+
 
     login() {
         this.loginButton.click()
     }
+
+    setLolcale() {
```
Comment (mdp0) [inline ui-tests/pages/landingPage.page.js:16]: Lolcale instead of locale :)

**Review by sameendph — APPROVED** [review]
Approving

File: ui-tests/pages/authenticationPage.page.js (hunk 1 above)
Comment (sameerapurini) [reply ui-tests/pages/authenticationPage.page.js:12]: Done, thanks for suggesting :)

File: ui-tests/pages/landingPage.page.js (hunk 2 above)
Comment (sameerapurini) [reply ui-tests/pages/landingPage.page.js]: Done

**Review by mdp0 — APPROVED** [review]

**Pull request merged**